packages/core/src/bundle.ts

```diff
@@ -267,6 +269,17 @@ function makeBundleVisitor(
     };
   }
 
+  function replaceInlineRef(node: OasRef, resolved: ResolveResult<any>, ctx: UserContext) {
+    const newRefId = ctx.location.source.absoluteRef + '::' + node.$ref;
```

Comment:
Make this a helper function in `resolve.ts` module:

`function makeRefId(absoluteRef, pointer)`

Reply:
done